website/client/components/auth/registerLoginReset.vue

```diff
@@ -472,6 +491,21 @@ export default {
 
       this.login();
     },
+    calculateTextWidth () {
+      let socialButtons = document.getElementById('social-buttons-row');
+      let facebookWidth = socialButtons.children[0].textContent.length;
+      let googleWidth = socialButtons.children[1].textContent.length;
+      if (facebookWidth < 15 && googleWidth < 15) {
+        return;
+      } else {
+        return this.changeButtonWidth(
+          Math.max(facebookWidth, googleWidth), socialButtons);
+      }
+    },
+    changeButtonWidth (width, div) {
+      div.style.marginRight = `-${(width - 15) / 3}em`;
```

Comment:
can the same thing we done using 100% - 15 instead of width?

Reply:
would this be ok?
`div.style.marginRight = 'calc(-(100% - 15)/3)em';`

Reply:
It should!